github/util.py

```diff
@@ -373,6 +373,16 @@ def from_githubrepobranch(
             default_branch=githubrepobranch.branch(),
         )
 
+    @staticmethod
+    def from_github_access(
```

Comment:
I would have preferred a util-function that would turn a github-access into a github-cfg (also below)